File: go-client-sdk/pkg/models/operations/get2.go
package operations

type Get2QueryParams struct {
	Foo string `queryParam:"style=form,explode=true,name=foo"`
}

type Get2Request struct {
	QueryParams Get2QueryParams
}

type Get2200ApplicationJSON struct {
	Args    map[string]interface{} `json:"args,omitempty"`
	Headers map[string]interface{} `json:"headers,omitempty"`
	Origin  *string                `json:"origin,omitempty"`
	URL     *string                `json:"url,omitempty"`
}

type Get2Response struct {
	ContentType                  string
	StatusCode                   int64
	Get2200ApplicationJSONObject *Get2200ApplicationJSON
}

File: node-client-sdk/src/sdk/models/operations/get2.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";


export class Get2QueryParams extends SpeakeasyBase {
  @Metadata({ data: "queryParam, style=form;explode=true;name=foo" })
  foo: string;
}


export class Get2Request extends SpeakeasyBase {
  @Metadata()
  queryParams: Get2QueryParams;
}


export class Get2200ApplicationJson extends SpeakeasyBase {
  @Metadata({ data: "json, name=args" })
  args?: Map<string, unknown>;

  @Metadata({ data: "json, name=headers" })
  headers?: Map<string, unknown>;

  @Metadata({ data: "json, name=origin" })
  origin?: string;

  @Metadata({ data: "json, name=url" })
  url?: string;
}


export class Get2Response extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;

  @Metadata()
  get2200ApplicationJsonObject?: Get2200ApplicationJson;
}
